apps/condo/domains/organization/integrations/sbbol/sync/syncPaymentRequestsForSubscriptions.js

```diff
@@ -6,46 +6,63 @@ const { SbbolFintechApi } = require('../SbbolFintechApi')
 const { SBBOL_YEARLY_SUBSCRIPTION_PRICE, SUBSCRIPTION_TYPE, SUBSCRIPTION_PAYMENT_STATUS, SUBSCRIPTION_PAYMENT_CURRENCY } = require('@condo/domains/subscription/constants')
 const conf = process.env
 const SBBOL_CONFIG = conf.SBBOL_CONFIG ? JSON.parse(conf.SBBOL_CONFIG) : {}
-const { debugMessage } = require('../common')
+const { logger: baseLogger } = require('../common')
 const { dvSenderFields, BANK_OPERATION_CODE } = require('../constants')
 const { SBBOL_OFFER_ACCEPT_IS_NULL_ERROR, SBBOL_OFFER_ACCEPT_HAS_INCORRECT_PAYER_REQUISITES_ERROR } = require('@condo/domains/subscription/constants/errors')
 
+const logger = baseLogger.child({ module: 'syncPaymentRequestsForSubscriptions' })
+
 // TODO(antonal): Add 3 days for trial subscription when payment is emitted to compensate payment processing lag and subsequent in service unavailability
 
 const postPaymentRequestFor = async (subscription, fintechApi) => {
     const { keystone: context } = await getSchemaCtx('ServiceSubscriptionPayment')
-    debugMessage('Start postPaymentRequestsFor', subscription)
-    debugMessage(`Processing ServiceSubscription(type="sbbol", id="${subscription.id}")`)
+    logger.info({ message: 'Start postPaymentRequestsFor', subscription })
 
     if (!subscription.sbbolOfferAccept) {
-        console.error(`${SBBOL_OFFER_ACCEPT_IS_NULL_ERROR} Cannot obtain requisites for payer from latest ServiceSubscription`)
-        debugMessage('Abort postPaymentRequestsFor')
+        logger.error({
+            message: 'Latest ServiceSubscription does not have information about offer accept',
+            error: SBBOL_OFFER_ACCEPT_IS_NULL_ERROR,
+            subscription,
+        })
         return
     }
 
     const { payerAccount, payerBankBic, payerBankCorrAccount, payerInn, payerName } = subscription.sbbolOfferAccept
 
     if (!payerAccount || !payerBankBic || !payerBankCorrAccount || !payerInn || !payerName) {
-        console.error(SBBOL_OFFER_ACCEPT_HAS_INCORRECT_PAYER_REQUISITES_ERROR, subscription.sbbolOfferAccept)
-        debugMessage('Abort postPaymentRequestsFor')
+        logger.error({
+            message: 'Latest ServiceSubscription has insufficient data in offer accept',
+            error: SBBOL_OFFER_ACCEPT_HAS_INCORRECT_PAYER_REQUISITES_ERROR,
+            sbbolOfferAccept: subscription.sbbolOfferAccept,
+        })
+        logger.info({ message: 'Abort postPaymentRequestsFor', subscription })
```

Comment:
it's better to remove it and add the subscription context inside the error line ^^